test/models/cart_test.rb

```diff
@@ -14,4 +14,75 @@ class CartTest < ActiveSupport::TestCase
     assert_equal({ "1" => 2, "2" => 1}, cart.contents)
   end
 
+  test 'can minus_item from cart' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.